fulp_modules/features/mobs/pony.dm

```diff
@@ -0,0 +1,47 @@
+/mob/living/simple_animal/hostile/pony
+	name = "Pony"
+	desc = "Is that a fucking Pony?"
```

Comment:
```suggestion
	desc = "Is that a fucking pony?"
```

Reply:
unless mlp has some weird capitalization related to the word "pony" that I don't know about